Sources/SafeDICore/Generators/ScopeGenerator.swift

```diff
@@ -249,33 +249,29 @@ actor ScopeGenerator {
         let propertiesToGenerate = Set(propertiesToGenerate.compactMap(\.property))
         for propertyToGenerateNode in orderedPropertiesToGenerate {
             let hasDependenciesGeneratedByCurrentScope = !propertyToGenerateNode
-                .value
```

Comment:
we no longer need this because we've implemented `@dynamicMemberLookup` on `List`